docs/design/hbar-limiter.md

```diff
@@ -19,6 +19,10 @@
       - [HBar Allocation Strategy](#hbar-allocation-strategy)
         - [Metrics to Track](#metrics-to-track)
         - [Allocation Algorithm](#allocation-algorithm)
+  - [Configurations](#configurations)
+    - [Pre-populating the Redis Cache with Spending Plans for Supported Projects and Partner Projects](#pre-populating-the-redis-cache-with-spending-plans-for-supported-projects-and-partner-projects)
```

Comment:
Yes, internally the repository classes use the `CacheService`, so `LRUCache` is used by default if Redis is not configured.

I will modify this to say just `Cache` instead of `Redis Cache`.